web-api/src/error/application.rs

```diff
@@ -122,116 +106,3 @@ pub(crate) struct Unimplemented {
 }
 
 impl_application_error!(Unimplemented => INTERNAL_SERVER_ERROR);
-
```

Comment:
All of this is no longer needed with the api call context in task local storage, we could also later one start using `#[get("/path/")]` from actix_web.